### PR TITLE
User metric fixes

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR fix user metrics not being initialized and some other undetected bugs and provides default metrics for existing users (migration).

Also fix a local dependant test on Me API

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
   [#799](https://github.com/opendatateam/udata/pull/799)
 - Some minor tuning on generic references
   [#801](https://github.com/opendatateam/udata/pull/801)
+- Fix user default metrics not being set
```
Comment (davidbgk) [inline CHANGELOG.md]: Proposition: adding a `[migration]` or something in this vein?

File: udata/migrations/2017-02-25-missing-user-metrics.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*
+ * Set default metric values on users
```
Comment (davidbgk) [inline udata/migrations/2017-02-25-missing-user-metrics.js]: Here we set new defaults but we do not compute metrics for all users again, what about current erroneous ones (if any)?

Comment (noirbizarre) [inline udata/migrations/2017-02-25-missing-user-metrics.js]: This PR doesn't handle the existing values, only default ones.
Existing values are already set and working (excluding those not being updated after transfer, see #810)

Comment (davidbgk) [inline udata/migrations/2017-02-25-missing-user-metrics.js]: Alright.

**Pull request merged**